Raspbian - cannot find function `parse_crate` in module `syn`

Hi, I am trying to pip3 install snips-nlu on my Raspberry.
I installed Rust and setuptools_rust but I can't figure out how to get this parse_crate function to work.

Any clue ?

``` Compiling cbindgen v0.4.3
  error[E0425]: cannot find function `parse_crate` in module `syn`
     --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/cbindgen-0.4.3/src/bindgen/parser.rs:167:30
      |
  167 |                 let i = syn::parse_crate(&s)
      |                              ^^^^^^^^^^^ not found in `syn`

  error[E0425]: cannot find function `parse_crate` in module `syn`
     --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/cbindgen-0.4.3/src/bindgen/parser.rs:281:30
      |
  281 |                 let i = syn::parse_crate(&s).map_err(|msg| {
      |                              ^^^^^^^^^^^ not found in `syn`

  error[E0277]: the trait bound `str: std::marker::Sized` is not satisfied
     --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/cbindgen-0.4.3/src/bindgen/parser.rs:281:55
      |
  281 |                 let i = syn::parse_crate(&s).map_err(|msg| {
      |                                                       ^^^ `str` does not have a constant size known at compile-time
      |
      = help: the trait `std::marker::Sized` is not implemented for `str`
      = note: all local variables must have a statically known size

  error: aborting due to 3 previous errors

  error: Could not compile `cbindgen`.
  warning: build failed, waiting for other jobs to finish...
  error: build failed
  error: cargo failed with code: 101```

--- a/setup.py
+++ b/setup.py
@@ -25,7 +25,7 @@ required = [
     "sklearn-crfsuite==0.3.6",
     "semantic_version==2.6.0",
     "snips_nlu_utils==0.6.0",
-    "snips_nlu_ontology==0.53.2",
+    "snips_nlu_ontology==0.54.0",
     "num2words==0.5.6"
 ]
 

